yb-voyager/src/query/queryparser/helpers_struct.go

```diff
@@ -264,3 +268,69 @@ func IsDDL(parseTree *pg_query.ParseResult) (bool, error) {
 	//Not Full-proof as we don't have all DDL types but atleast we will skip all the types we know currently
 	return !ok, nil
 }
+
+func IsJsonbType(node *pg_query.Node, jsonbColumns []string, jsonbFunctions []string) bool {
+	switch {
+	case node.GetColumnRef() != nil:
+		/*
+			SELECT numbers[1] AS first_number
+			FROM array_data;
+			{a_indirection:{arg:{column_ref:{fields:{string:{sval:"numbers"}}  location:69}}
+			indirection:{a_indices:{uidx:{a_const:{ival:{ival:1}  location:77}}}}}}  location:69}}
+		*/
+		_, col := GetColNameFromColumnRef(node.GetColumnRef().ProtoReflect())
+		if slices.Contains(jsonbColumns, col) {
+			return true
+		}
+
+	case node.GetTypeCast() != nil:
+		/*
+			SELECT ('{"a": {"b": {"c": 1}}}'::jsonb)['a']['b']['c'];
+			{a_indirection:{arg:{type_cast:{arg:{a_const:{sval:{sval:"{\"a\": {\"b\": {\"c\": 1}}}"}  location:280}}
+			type_name:{names:{string:{sval:"jsonb"}}  typemod:-1  location:306}  location:304}}
+		*/
+		typeCast := node.GetTypeCast()
+		typeName, _ := GetTypeNameAndSchema(typeCast.GetTypeName().GetNames())
```

Comment:
GetTypeNameAndSchema need not be exported anymore because you moved the entire function to queryparser? Same probably for a few other functions that you decided to export. Pls check.